manifest.ini

```diff
@@ -1,6 +1,20 @@
 [stage]
 reader_server = https://kinto.stage.mozaws.net/v1
 writer_server = https://kinto-writer.stage.mozaws.net/v1
+server = https://kinto.stage.mozaws.net/v1/
+version_fields = source, name, commit, version, build
+heartbeat_fields = storage, cache, permission
 
-[prod]
+[settings-prod]
 reader_server = https://firefox.settings.services.mozilla.com/v1/
```

Comment:
They are different values used for different tests and I don't see the harm in the duplication.